dashboard/package.json

```diff
@@ -23,21 +23,24 @@
     "@types/vfile": "^4.0.0",
     "@types/vfile-message": "^2.0.0",
     "@types/ws": "^6.0.0",
-    "@types/yaml": "^1.0.2",
+    "@types/yaml": "^1.9.7",
     "ajv": "^6.10.2",
     "axios": "^0.19.0",
-    "connected-react-router": "^4.5.0",
+    "connected-react-router": "^6.8.0",
```

Comment:
This change requires some migration, as per https://github.com/supasate/connected-react-router/blob/master/FAQ.md#how-to-migrate-from-v4-to-v5v6